### PR TITLE
Revert previous change for offline event overview

### PR DESCRIPTION
**Pull request by cibelius**
## Description

Core PR https://github.com/opencrvs/opencrvs-core/pull/11625

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: e2e/testcases/events-rest-api/create-and-notify-event.spec.ts [hunk 1]
```diff
@@ -990,10 +990,6 @@ test.describe('Events REST API', () => {
         .getByText(await formatV2ChildName({ 'child.name': childName }))
         .click()
 
-      await expect(page.locator('#content-name')).toHaveText(
```
Comment (cibelius) [inline e2e/testcases/events-rest-api/create-and-notify-event.spec.ts:993]: This assertion was causing flakyness, remove it

**Review by makelicious — APPROVED** [review]
